assets/src/tower_config/tower.yml.tpl

```diff
@@ -122,6 +122,17 @@ tower:
   ephemeral:
     duration: 8h                                            # Example values: 1m, 8h, 1d. Default: 8h
 
+  # Current as of v24.3, default size is 20480. This can result in errors for some particularly
+  # verbose pipelines: `{"message":"ParamsText field too big - Max allowed size 20480 characters"}`
+  #
+  # Corresponding database field can accommodate max ~4GB payload, but we recommend this value
```

Comment:
Database is not the only limit, also web server does not allow payload > ~ 10MB.

Reply:
Thanks @pditommaso -- I had a further callout below re: the size (I had been told it was 11MB). Maybe I should just remove the max db size entirely?

Reply:
> Maybe I should just remove the max db size entirely?

make sense